.travis.yml

```diff
@@ -0,0 +1,85 @@
+    #Core - Network Test
+      - echo ">>> Core - Network Test ... >>>"
+      - cd $TRAVIS_BUILD_DIR/build/olp-cpp-sdk-core/tests/network/$BUILD_TYPE
+      - "./olp-core-network-test.exe --gtest_output=xml || echo 'Core - Network Test failed'"
+      - "[ -f 'test_detail.xml' ] && cp test_detail.xml $TRAVIS_BUILD_DIR/test_results/olp-core-network-test.xml"
+    #Core - Client Test
+      - echo ">>> Core - Client Test ... >>>"
+      - cd $TRAVIS_BUILD_DIR/build/olp-cpp-sdk-core/tests/olpclient/$BUILD_TYPE
+      - "./olpclient-test.exe --gtest_output=xml --endpoint='https://account.api.here.com/oauth2/token'
+                             --catalog='hrn:here:data:::hereos-internal-test-v2'
+      || echo 'Core - Client Test failed'"
+      - "[ -f 'test_detail.xml' ] && cp test_detail.xml $TRAVIS_BUILD_DIR/test_results/olpclient-test.xml"
+    #Data Service Read Test
+      - echo ">>> Data Service Read Test ... >>>"
+      - cd $TRAVIS_BUILD_DIR/build/olp-cpp-sdk-dataservice-read/test/unit/$BUILD_TYPE
+      - "./olp-dataservice-read-test.exe --gtest_output=xml --endpoint='https://account.api.here.com/oauth2/token' 
+                                        --catalog='hrn:here:data:::hereos-internal-test-v2' 
+                                        --appid=$DATASERVICE_READ_APPID 
+                                        --secret=$DATASERVICE_READ_SECRET
+      || echo 'Data Service Read Test failed'"
+      - "[ -f 'test_detail.xml' ] && cp test_detail.xml $TRAVIS_BUILD_DIR/test_results/olp-dataservice-read-test.xml"
+    #Data Service Write Test
+      - echo ">>> Data Service Write Test ... >>>"
+      - cd $TRAVIS_BUILD_DIR/build/olp-cpp-sdk-dataservice-write/test/$BUILD_TYPE
+      - "./olp-dataservice-write-test.exe --gtest_output=xml
+                                         --endpoint=https://account.api.here.com/oauth2/token
+                                         --catalog=hrn:here:data:::olp-cpp-sdk-ingestion-test-catalog
+                                         --layer=olp-cpp-sdk-ingestion-test-stream-layer
+                                         --layer2=olp-cpp-sdk-ingestion-test-stream-layer-2
+                                         --layer-sdii=olp-cpp-sdk-ingestion-test-stream-layer-sdii
+                                         --versioned-layer=olp-cpp-sdk-ingestion-test-versioned-layer
+                                         --volatile-layer=olp-cpp-sdk-ingestion-test-volatile-layer
+                                         --index-layer=olp-cpp-sdk-ingestion-test-index-layer
+                                         --appid=$DATASERVICE_WRITE_APPID
+                                         --secret=$DATASERVICE_WRITE_SECRET
+      || echo 'Data Service Write Test failed'"
+      - "[ -f 'test_detail.xml' ] && cp test_detail.xml $TRAVIS_BUILD_DIR/test_results/olp-dataservice-write-test.xml"
+      - echo "tests finished"
```

Comment:
We should put all of this into a `scripts` folder so that we dont need to change Travis config every time when we add ne test or new parameters.